### PR TITLE
Create custom price feed for WBTC

### PR DESCRIPTION
**Pull request by kevincheng96**
This PR creates a custom price feed for WBTC / USD by reading from both the WBTC / BTC and BTC / USD price feeds. This will allow the cUSDCv3 market to get a more accurate price for WBTC.

We also organize the contracts repo by moving all the custom price feed contracts into a separate `pricefeeds` subdirectory.

### CODE REVIEW

File: contracts/pricefeeds/WBTCPriceFeed.sol [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        uint80 answeredInRound
+    ) {
+        (uint80 roundIdA, int256 WBTCToBTCPrice, uint256 startedAtA, uint256 updatedAtA, uint80 answeredInRoundA) = AggregatorV3Interface(WBTCToBTCPriceFeed).latestRoundData();
+        (uint80 roundIdB, int256 BTCToUSDPrice, uint256 startedAtB, uint256 updatedAtB, uint80 answeredInRoundB) = AggregatorV3Interface(BTCToUSDPriceFeed).latestRoundData();
+
+        // We use return the round data of the BTC / USD price feed because of its shorter heartbeat (1hr vs 24hr)
+        if (WBTCToBTCPrice <= 0 || BTCToUSDPrice <= 0) return (roundIdB, 0, startedAtB, updatedAtB, answeredInRoundB);
+
+        int256 price = WBTCToBTCPrice * BTCToUSDPrice * priceFeedScale / combinedScale;
+        return (roundIdB, price, startedAtB, updatedAtB, answeredInRoundB);
```
Comment (kevincheng96) [inline contracts/pricefeeds/WBTCPriceFeed.sol:78]: Kind of odd that we have to choose one of the two underlying price feeds to use for the round data. But I figure it's better to use the one with the shorter heartbeat.

File: contracts/pricefeeds/WBTCPriceFeed.sol [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    uint public constant override version = 1;
+
+    /// @notice Description of the price feed
+    string public constant override description = "Custom price feed for WBTC / USD";
+
+    /// @notice Number of decimals for returned prices
+    uint8 public immutable override decimals;
+
+    /// @notice Chainlink WBTC / BTC price feed
+    address public immutable WBTC_TO_BTC_PRICE_FEED;
```
Comment (scott-silver) [inline contracts/pricefeeds/WBTCPriceFeed.sol:27]: This is a nit, but generally we've reserved `UPPER_CASE` for constants, not immutables.

The [style guide](https://docs.soliditylang.org/en/latest/style-guide.html#constants) that OZ referenced only specifies this for constants.

Comment (kevincheng96) [reply contracts/pricefeeds/WBTCPriceFeed.sol]: Good point! I reverted the `UPPER_CASE` change since there isn't a recommended style guide for immutables.

**Comment by kevincheng96** [comment]
Final [audit report](https://gist.github.com/kacperrams/7242067e60392e4c96f9a01ba81a7026) from OZ

**Review by se6799 — COMMENTED** [review]
Yelp

**Pull request merged**